Sign_In_Page.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import Secure 
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.signInPage_url)
+    
+
+    def click_sign_in_button(self):
+        for cookie in Secure.cookies_coinbase: self.driver.add_cookie(cookie)
+        self.driver.find_element_by_id('signin_button').click()
+
+    def email_field(self): return self.driver.find_element_by_id('email')
```

Comment:
For all these, do the same as with the signin button. Pull the locator out.

Also, build these consistently. In your homepage page object, you didn't make a separate find_element method, you simply made a click_ method and found the element in there. If that's how you want to do it, do it that way. But if you want to build a find_element() method for each element, do that. But don't mix.

I don't mind which you do, there are pros and cons to each.